SWLOR.Game.Server/Service/SkillService/SkillType.cs

```diff
@@ -20,7 +20,8 @@ public enum SkillType
             true,
             "Ability to use one-handed weapons like vibroblades, finesse vibroblades, and lightsabers.",
             true,
-            false)]
+            false,
+            CombatPointCategory.Weapons)]
```

Comment:
Super nit picky, but try to use the singular to be consistent. Swap this from Weapons to Weapon please.